apps/src/templates/sectionProgress/ProgressBox.jsx

```diff
@@ -60,17 +74,24 @@ export default class ProgressBox extends Component {
       ...styles.lessonNumber,
       color: perfect ? color.white : color.charcoal,
     };
+
     return (
       <div style={boxWithBorderStyle}>
         {this.props.lessonNumber && (
           <div style={lessonNumberStyle}>{this.props.lessonNumber}</div>
         )}
-        <div style={incompleteLevels} />
-        <div style={imperfectLevels} />
-        <div
-          className={'uitest-perfect-bar'}
-          style={lessonIsAllAssessment ? assessmentLevels : perfectLevels}
-        />
+        {viewed ? (
+          <div style={viewedStyle} />
+        ) : (
+          <div>
```

Comment:
```suggestion
          <>
```
I'm also fine as is, as long as we can confirm that no existing use for ProgressBox is changed by adding a div.